src/screens/Requests/Requests.tsx

```diff
@@ -239,22 +239,34 @@ const Requests = (): JSX.Element => {
   };
 
   /* istanbul ignore next */
-  const handleSearchByName = async (e: any): Promise<void> => {
+  const handleSearch = async (value: string): Promise<void> => {
```

Comment:
please remove /* istanbul ignore next */ comment and resubmit the PR.

Reply:
Sure, I will remove them.